electron/src/renderer/components/Simulator.vue

```diff
@@ -191,6 +213,17 @@
               <font-awesome-icon icon="horse" size="2x" class="fa-button fa-fast"/>
             </button>
           </fieldset>
+          <fieldset class="view">
+            <legend>
+              <translate>View</translate>
+            </legend>
+            <button v-on:click="zoomPage" :title="$gettext('Zoom page ([)')">
+              <font-awesome-icon icon="search-minus" size="2x" class="fa-button"/>
+            </button>
+            <button v-on:click="zoomSelection" :title="$gettext('Zoom selection (])')">
```

Comment:
I'm not sure "selection" is the right word here, how about "Zoom design"?

Also I really like the addition of this View section.  I wonder if we should also add buttons for zooming in and out incrementally.  Not sure if there are any good font-awesome icons left though :)

Reply:
I am cool with "Zoom design", changed it.
I agree with adding more nav/zoom options, currently it would be hard to zoom if you were using a tablet or a mouse without a scroll wheel.

Reply:
Oh, is it?  I thought svg.zoom.js handled pinch gestures...

Reply:
Oh yeah trackpads are fine, I was mostly thinking about when using my Wacom tablet, it's a bit awkward. 
Adding a hot key to switch between pan and zoom would probably be a pretty easy solution.

Reply:
Oh _that_ kind of tablet!